### ISSUE TITLE
module analysis only works for relative imports

### ISSUE THREAD

**Issue by michaelangeloio** [michaelangeloio/does-it-throw#47]
Currently, we are only resolving relative imports:

https://github.com/michaelangeloio/does-it-throw/blob/99fda183a7ca813cbb5f5434f429cd79b594f139/crates/does-it-throw-wasm/src/lib.rs#L136

It would be nice to perform the same analysis in Typescript alias paths. I am not sure about node_modules, but I am open to ideas.

**Comment by boian-ivanov** [comment]
This would be godsend, as most of the projects I work with are using aliases, so even if they're not read from the `tsconfig.json`, but manually added it would be a huge boost of understanding if an imported method throws 🎉

**Comment by djolejov** [comment]
Yes! I second this, aliases are always used in projects I work with.
